### DISCUSSION TITLE
Packages/com.unity.render-pipelines.core/api/UnityEngine.Rendering.ProfilingScope

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#246] [Notes]
# Packages/com.unity.render-pipelines.core/api/UnityEngine.Rendering.ProfilingScope

https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@latest/index.html?subfolder=%2Fapi%2FUnityEngine.Rendering.ProfilingScope.html

**Comment by vertxxyz** [comment]
I've found if you don't call `Clear` on the CommandBuffer you're passing to the scope, then it may have a recursive freakout when you then try to use the Frame Debugger.
